target/linux/ramips/dts/mt7620n_zyxel_lte-3302-m432.dts

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ * ZyXEL LTE3302-M432 Board Description
+ * Copyright 2022 Pawel Dembicki <paweldembicki@gmail.com>
+ */
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "zyxel,lte-3302-m432", "ralink,mt7620n-soc";
+	model = "ZyXEL LTE3302-M432";
+
+	aliases {
+		led-boot = &led_signal_green;
+		led-failsafe = &led_signal_red;
+		led-running = &led_signal_green;
+		led-upgrade = &led_signal_red;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led-0 {
+		    function = LED_FUNCTION_LAN;
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio1 14 GPIO_ACTIVE_HIGH>;
+		};
+
+		led-1 {
+		    function = LED_FUNCTION_WAN;
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+		};
+
+		led_signal_green: led-2 {
+			label = "green:signal";
+            function = LED_FUNCTION_INDICATOR;
```

Comment:
There are multiple whitespace errors in this file

Reply:
Fixed.